DigitalLearningSolutions.Data/DataServices/UserDataService/UserDataService.cs

```diff
@@ -58,6 +58,10 @@ int[] ids
 
         public int GetDelegateCountWithAnswerForPrompt(int centreId, int promptNumber);
         public void DeleteAllAnswersForPrompt(int centreId, int promptNumber);
+
+        public int GetAnswerCountForCourseAdminField(int customisationId, int promptNumber);
+
+        public void DeleteAllAnswersForAdminField(int centreId, int promptNumber);

```

Comment:
Already mentioned this below for one of them, but I think both should live elsewhere :)

Reply:
moved to the appropriate file